examples/style.yaml

```diff
@@ -55,14 +55,14 @@ jobs:
             ${{ runner.os }}-
 
       - name: Style
-        run: styler::style_pkg(filetype = c(".R", ".Rmd", ".Rmarkdown", ".Rnw"))
+        run: styler::style_pkg()
         shell: Rscript {0}
 
       - name: Commit and push changes
         run: |
           git config --local user.name "$GITHUB_ACTOR"
           git config --local user.email "$GITHUB_ACTOR@users.noreply.github.com"
-          git add R/\*
+          git add \*.[rR] \*.[qrR]md \*.[rR]markdown \*.[rR]nw \*.[rR]profile
```

Comment:
This throws an error, if one of those extensions is not available in the repo (which is quite likely). 😫

Reply:
git add --ignore-errors throws the [same error](https://github.com/ropensci/opencage/actions/runs/4026891715/jobs/6921953697#step:9:21).